app/controllers/coronavirus_form/carry_supplies_controller.rb

```diff
@@ -19,7 +19,7 @@ def submit
         format.html { render controller_path, status: :unprocessable_entity }
       end
     elsif session[:check_answers_seen]
-      session[:essential_supplies] = @form_responses[:essential_supplies]
+      session[:carry_supplies] = @form_responses[:carry_supplies]
```

Comment:
Ah! You spotted it. This should probably be fixed up to the previous commit.

Reply:
Looks like I had fixed this in the wrong commit 🙊